tests/minifier.js

```diff
@@ -1962,6 +1962,22 @@
     equal(minify(input, options), output);
   });
 
+  test('ignore synthetic tags', function() {
+    equal(minify('<p id=""class=""title="">x', { ignoreSyntheticTags: true }), '<p id="" class="" title="">x');
```

Comment:
Thanks. Yeah, it's not really intuitive to me what `syntheticTags` mean. Could we perhaps say `autoGenerated` or `autoClosing`? Or is "synthetic" the actual term from the spec?